docs/guide-helm.md

```diff
@@ -17,7 +17,7 @@ This document will show you how to use `Helm` for `NSM` installation.
 Installing NSM with helm in the `nsm-system` namespace is as easy as:
 
 ```bash
-$ helm install deployments/helm/nsm
+$ helm install nsm deployments/helm/nsm
 ```
 
 *Note: in case of `Error: no available release name found` do (according to [issue](https://github.com/helm/helm/issues/4412)):*
```

Comment:
This whole section can be probably removed, right?

Reply:
You're right, thanks!
- [ ] remove tiller issue troubleshooting